cpp/foxglove-websocket/include/foxglove/websocket/server.hpp

```diff
@@ -31,11 +32,17 @@ struct ChannelWithoutId {
   std::string encoding;
   std::string schemaName;
   std::string schema;
+
+  bool operator==(const ChannelWithoutId& other) const {
+    return topic == other.topic && encoding == other.encoding && schemaName == other.schemaName &&
```

Comment:
just do a 
```
bool operator==(ChannelWithoutId const& rhs) const = default;
```
It should be C++11 and the compiler generates just that.

Reply:
Hmm, seems this is a c++20 feature? never heard of it before :) https://en.cppreference.com/w/cpp/language/default_comparisons

Reply:
I could have sworn that I used it before, but apparently not... too bad 😞